modules/page-memory/src/main/java/org/apache/ignite/internal/pagememory/datapage/DataPageReader.java

```diff
@@ -54,129 +49,49 @@ public DataPageReader(PageMemory pageMemory, int groupId, IoStatisticsHolder sta
     }
 
     /**
-     * Returns a row by link. To get the row bytes, more than one pages may be traversed (if the corresponding row
-     * was fragmented when stored).
+     * Traverses page memory starting at the given link. At each step, reads the current data row and feeds it to the given
```

Comment:
I would avoid "data row" mentioning in this context. I tried to abstract from pages content.

Reply:
How would you replace it?